luigi/date_interval.py

```diff
@@ -14,6 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+"""
+luigi.date_interval provides convenient classes for date algebra.
+There is a corresponding :class:`luigi.parameter.DateIntervalParameter` that you can use to parse date intervals.
+Everything uses ISO 8601 notation, i.e. YYYY-MM-DD for dates, etc.
+
+Example:
+
+.. code-block:: python
```

Comment:
this can be written as (the default language of sphinx is python)

```rst
Example::

    class MyTask(luigi.Task):
        date_interval = luigi.DateIntervalParameter()
```